zulip_bots/zulip_bots/bots/converter/converter.py

```diff
@@ -83,14 +87,14 @@ def get_bot_converter_response(message, bot_handler):
                     exponent -= exp
                     unit_to = unit_to[len(key):]
 
-            uf_to_std = utils.UNITS.get(unit_from, False)
-            ut_to_std = utils.UNITS.get(unit_to, False)
+            uf_to_std = utils.UNITS.get(unit_from, [])  # type: Any
```

Comment:
If `type: any` is not specified, mypy gives an error: 
![conversion-error](https://user-images.githubusercontent.com/22359229/33801626-6ba5b40e-dd15-11e7-9f40-1758dbdbe9c5.png)

Is specifying that okay?

Reply:
Hmm.. weird, I don't get any errors when I pulled your PR and ran the tests locally. @neiljp: Any ideas? :)

Reply:
@KaiUsesThis could you try running mypy again and make sure you are on the correct branch? :)

Reply:
@KaiUsesThis: Maybe try `# type: List[Any]`, since that seems to be the correct annotation! :)

Reply:
Ran mypy again, no errors when ran with `type: any`, but there were errors when `type: any` was removed. I am on the correct branch

Reply:
@eeshangarg Okay, I've changed the type to `List[Any]`